sw/device/tests/chip_power_idle_load.c

```diff
@@ -0,0 +1,351 @@
+      .blink_parameter_x = 0,
+      .blink_parameter_y = 0,
+  };
+  static const dif_pwm_channel_t kPwmChannel[PWM_PARAM_N_OUTPUTS] = {
+      kDifPwmChannel0, kDifPwmChannel1, kDifPwmChannel2,
+      kDifPwmChannel3, kDifPwmChannel4, kDifPwmChannel5,
+  };
+  // Duty cycle values
+  static volatile const uint16_t kPwmDutycycle[PWM_PARAM_N_OUTPUTS] = {
+      6, 11, 27, 8, 17, 7,
+  };
+
+  static const dif_pinmux_index_t kPinmuxMioOut[PWM_PARAM_N_OUTPUTS] = {
+      kTopEarlgreyPinmuxMioOutIob10, kTopEarlgreyPinmuxMioOutIob11,
+      kTopEarlgreyPinmuxMioOutIob12, kTopEarlgreyPinmuxMioOutIoc10,
+      kTopEarlgreyPinmuxMioOutIoc11, kTopEarlgreyPinmuxMioOutIoc12,
+  };
+  static const dif_pinmux_index_t kPinmuxOutsel[PWM_PARAM_N_OUTPUTS] = {
+      kTopEarlgreyPinmuxOutselPwmAonPwm0, kTopEarlgreyPinmuxOutselPwmAonPwm1,
+      kTopEarlgreyPinmuxOutselPwmAonPwm2, kTopEarlgreyPinmuxOutselPwmAonPwm3,
+      kTopEarlgreyPinmuxOutselPwmAonPwm4, kTopEarlgreyPinmuxOutselPwmAonPwm5,
+  };
+
+  CHECK_DIF_OK(dif_pwm_configure(&pwm, config_));
+
+  // Confugure each of the PWM channels:
+  dif_pwm_channel_config_t channel_config_ = default_ch_cfg_;
+  for (int i = 0; i < PWM_PARAM_N_OUTPUTS; ++i) {
+    CHECK_DIF_OK(
+        dif_pwm_channel_set_enabled(&pwm, kPwmChannel[i], kDifToggleDisabled));
+    channel_config_.duty_cycle_a = kPwmDutycycle[i];
+    CHECK_DIF_OK(
+        dif_pwm_configure_channel(&pwm, kPwmChannel[i], channel_config_));
+    CHECK_DIF_OK(
+        dif_pwm_channel_set_enabled(&pwm, kPwmChannel[i], kDifToggleEnabled));
+  }
+
+  // Enable all PWM channels
+  CHECK_DIF_OK(dif_pwm_phase_cntr_set_enabled(&pwm, kDifToggleEnabled));
+
+  // PINMUX
+  for (int i = 0; i < PWM_PARAM_N_OUTPUTS; ++i) {
+    CHECK_DIF_OK(
+        dif_pinmux_output_select(&pinmux, kPinmuxMioOut[i], kPinmuxOutsel[i]));
+  }
+
+  LOG_INFO("SW_INFO: PWM active");
+
+  // OTP
+
+  static const dif_otp_ctrl_config_t otp_ctrl_config = {
+      .check_timeout = UINT32_MAX,
+      .integrity_period_mask = 0x1,
+      .consistency_period_mask = 0x1,
+  };
+
+  CHECK_DIF_OK(dif_otp_ctrl_configure(&otp_ctrl, otp_ctrl_config));
+
+  LOG_INFO("SW_INFO: OTP periodic checks active");
+
+  // AON Timer - activate in Watchdog mode & IRQ
+  static const uint64_t kTimeTillBark = 1000;
+
+  CHECK_DIF_OK(dif_aon_timer_watchdog_stop(&aon_timer));
+  CHECK_DIF_OK(dif_aon_timer_clear_wakeup_cause(&aon_timer));
+  expected_isr_handler = wdog_irq_handler;
+  nmi_fired = false;
+  nmi_state = (dif_rv_core_ibex_nmi_state_t){0};
+  CHECK_DIF_OK(
+      dif_rv_core_ibex_enable_nmi(&rv_core_ibex, kDifRvCoreIbexNmiSourceWdog));
+  uint32_t count_cycles =
+      aon_timer_testutils_get_aon_cycles_from_us(kTimeTillBark);
+  aon_timer_testutils_watchdog_config(&aon_timer, count_cycles, UINT32_MAX,
+                                      false);
+
+  LOG_INFO("SW_INFO: AON Timer active");
+
+  LOG_INFO("SW_INFO: all H/W is active");
+  // indicate S/W is idle
+  test_status_set(0xff20);
```

Comment:
What is the intended usage model for setting this to an arbitrarily chosen value? 
The SW and SV "agree" on a limited set of flags to indicate what stage the SW is in. Setting this to values outside of that limited set has an undefined behavior.

Reply:
removed it